docs/source/notebooks/iv_pymc.ipynb

```diff
@@ -0,0 +1,4154 @@
+{
```

Comment:
<p>Not sure why but my brain took a while to parse loggdp as it is. Maybe have as maths to it's clearer we're talking about GDP.</p><p> </p><p>We could in fact have a custom graph using generic symbols, then the same graph but using the particular variable names of the example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Added this one: 
<img width="719" alt="image" src="https://github.com/pymc-labs/CausalPy/assets/10588167/a9ef145f-78fd-4188-8e4c-ca8375262974">
 and then: 
<img width="691" alt="image" src="https://github.com/pymc-labs/CausalPy/assets/10588167/85549920-baef-4d66-b243-3b4ca2ad72a1">

Also added a parameter recovery example with simulated data next to the generic graph. To emphasise the nature of the confounding.